### PR TITLE
Integrate usage of neptune graph SDK for schema queries and lambda logic

### PR DESCRIPTION
**Pull request by andreachild**
Introduce usage of the neptune graph (analytics) SDK in a couple scenarios: 1. during pipeline creation, if an axios request fails when querying the Neptune Analytics graph and falls back to SDK and 2. if the user has opted to use SDK for the generated lambda (as opposed to http). 

Summary of changes:
* added new dependency on client-neptune-graph version 3.662.0 
* created new lambda template which is used if the user specifies --output-resolver-query-sdk option
* set additional lambda environment variable for neptune db name which is required to execute queries using the neptune graph SDK
* added logic to fall back to neptune graph SDK if Axios request fails during pipeline creation (previous logic threw Error as the analytics SDK was not yet available)
* fixed function which retrieves graph summary to use neptune graph SDK if the neptune-type is neptune-graph (the summary endpoint path for neptune-db is not the same for neptune-graph)
* fixed CDK pipeline to only fetch cluster info if the type is neptune-db as it is not required for Neptune-graph (analytics)
* set isNeptuneIAMAuth to true if the neptune type is detected as neptune-graph
* introduced util.js for parsing functions that are used across multiple modules
* refactored function which had many params to use an object param instead for better readability
* fixed some typos which specified 'neptume' instead of 'neptune' 
* changed queryNeptune function to have default value of empty object for params
* change jest config from json to js to allow for globals to be set from environment variables
* changed test cases to reference <AIR_ROUTES_DB_HOST> and <AIR_ROUTES_DB_PORT values which are swapped at test runtime for the environment variable values for easier test configurability and to prevent future accidental commit of real URLs
* removed duplicated airports.source.schema.graphql file in test cases and changed to reference single file in parent directory
* fixed Case05/case02.json which I believe was accidentally changed from remove to create pipeline
* introduced new test case 7 which sets --output-resolver-query-sdk option

### CODE REVIEW

File: src/NeptuneSchema.js [hunk 1]
```diff
@@ -109,6 +122,31 @@ async function queryNeptuneSDK(query, params) {
     }
 }
 
+/**
+ * Queries neptune analytics graph using SDK (not to be used for neptune db).
+ */
+async function queryNeptuneGraphSDK(query, params = '{}') {
+    try {
+        const client = new NeptuneGraphClient({
```
Comment (Cole-Greer) [inline src/NeptuneSchema.js]: There is some non-trivial overhead to creating a new client for each query. I don't think this needs to be a blocker at the moment, however it should be noted that refactoring this to reuse a single client for the whole life of the pipeline should improve efficiency.

**Comment by Cole-Greer** [comment]
> added logic to fall back to neptune graph SDK if Axios request fails during pipeline creation (previous logic threw Error as the analytics SDK was not yet available)

I'm wondering if it would be preferable to try the SDK first. Unlike Neptune-DB, all versions of neptune-graph support queries via the SDK.

**Comment by andreachild** [comment]
> > added logic to fall back to neptune graph SDK if Axios request fails during pipeline creation (previous logic threw Error as the analytics SDK was not yet available)
> 
> I'm wondering if it would be preferable to try the SDK first. Unlike Neptune-DB, all versions of neptune-graph support queries via the SDK.

Next major version I want to remove the http option and at that point there will not be a fallback. I would rather wait until then to address this otherwise there will be different fallback logic for neptune-db (http then sdk) vs neptune-graph (sdk then http).

File: src/NeptuneSchema.js (hunk 1 above)
Comment (andreachild) [reply src/NeptuneSchema.js]: Changed both sdk clients to be created only once per pipeline.

**Review by Cole-Greer — APPROVED** [review]

**Pull request merged**